sample_annotator/ontology/package_checklist_normalizer.py

```diff
@@ -0,0 +1,132 @@
+# PyCharm complaining that argparse==1.4.0 is not satisfied
+# and that yaml is not listed in the project requirements
+# unsuccessful previous attempt to install nim plugin for code-formatting .cfg files
+# https://plugins.jetbrains.com/plugin/15128-nim
+import yaml
+import requests
+import pandas as pd
+from xml.etree import ElementTree
+import re
+from string import punctuation
+
+MULTI_WHITESPACE = re.compile(r"\s+")
+
+# put some caching mechanism in place
+
+# # get/show the most common env package values in harmonized_table.db ?
+# # created by https://github.com/INCATools/biosample-analysis
+# # data/env_package_counts_202108100944.tsv
+# select
+# 	env_package,
+# 	count(1)
+# from
+# 	biosample b
+# group by
+# 	env_package
+# order by
+# 	count(1) desc;
+
+# what to do with NULLs and empty strings?
+
+# examine https://github.com/cmungall/mixs-source with linkml methods to see what packages are allowed?
+#   or maybe just pyyaml if no fancy methods are required
+# env_package_enum is defined in in terms.yaml
+# and corresponds to EnvPackageDisplay in https://www.ncbi.nlm.nih.gov/biosample/docs/packages/?format=xml
+# ['air', 'built environment', 'host-associated', 'human-associated', 'human-skin', 'human-oral', 'human-gut',
+# 'human-vaginal', 'hydrocarbon resources-cores', 'hydrocarbon resources-fluids/swabs', 'microbial mat/biofilm',
+# 'misc environment', 'plant-associated', 'sediment', 'soil', 'wastewater/sludge', 'water']
+
+def get_envpacks_from_mixssource(mixssource_terms_url="https://raw.githubusercontent.com/cmungall/mixs-source/main"
```

Comment:
As convention (at least that I've seen in others' code) urls are first declared as a variable up top (maybe at line 13 below `MULTI_WHITESPACE`. So could we have line 13 as:
```
MIXS_SOURCE_TERMS_URL="https://raw.githubusercontent.com/cmungall/mixs-source/main"
```
and line 39 be `mixssource_terms_url=MIXS_SOURCE_TERMS_URL`

Reply:
agreed